recipe/meta.yaml

```diff
@@ -45,11 +44,12 @@ requirements:
     - rich >=6.0,!=12.0.0
     - scipy >=1.7.3,<2.0.0
     - screeninfo >=0.8.0,<0.9.0
-    - skia-pathops >=0.7.0,<0.8.0
+    - skia-pathops >=0.8.0.post1,<0.9.0
```

Comment:
Technically, this is a

```toml
skia-pathops = [
    { version = "^0.7.0", python = "<3.12" },
    { version = "^0.8.0.post1", python = ">=3.12" }
]
```

Not sure if this is representable in a noarch package.

Reply:
Maybe we can just change it to `>=0.7.0`? It should work right?

Reply:
If the CI builds and tests with python 3.12, `pip check` will probably fail but I guess it doesn't hurt to try.